.github/workflows/tests.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+name: Tests
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [8.1, 8.2]
+        symfony-version: ["^5.2", "^6.3"]
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          tools: composer
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: composer install
+      - name: Run test
+        run: vendor/bin/phpunit --configuration phpunit.xml --coverage-text
+      - name: Run CS fixer
+        run: vendor/bin/php-cs-fixer fix --dry-run
```

Comment:
Would it make sense to run this in it's own job?

Reply:
It would yeah, I just migrated the pipeline

I'll create a dedicated job 😉